cosmos/x/evm/plugins/block/genesis.go

```diff
@@ -26,21 +26,32 @@ import (
 	"pkg.berachain.dev/polaris/eth/core"
 )
 
-// InitGenesis performs genesis initialization for the evm module. It returns
-// no validator updates.
-func (p *plugin) InitGenesis(ctx sdk.Context, _ *core.Genesis) {
-	// TODO: IMPLEMENT
+// InitGenesis performs genesis initialization for the block plugin in the
+// evm module.
+func (p *plugin) InitGenesis(ctx sdk.Context, ethGen *core.Genesis) {
 	p.Prepare(ctx)
-	// p.StoreHeader(ethGen.ToBlock().Header())
```

Comment:
add a comment here saying that we are storing the genesis block. 

or maybe make an abstraction for readability.